### PR TITLE
update useful addresses

### PR DESCRIPTION
**Pull request by krofax**
Updated the useful address section, with more addresses listed on our block explorer.

### CODE REVIEW

File: docs/dev/building-on-zksync/useful-address.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ Here are some contract addresses that may be useful and/or of interest to those
 
 ## Mainnet contract addresses
 
+These are the list of mainnet contract addresses that deployed on zkSync Era network.
```
Comment (kailash-manasarovar) [inline docs/dev/building-on-zksync/useful-address.md:7]:
```suggestion
```

File: docs/dev/building-on-zksync/useful-address.md [hunk 2]
```diff
@@ -39,6 +41,7 @@ _Users should only interact with the token bridge contracts via [zkSync Bridge](
 
 ## Testnet contract addresses
 
+These are the list of testnet contract addresses that deployed on zkSync Era network.
```
Comment (kailash-manasarovar) [inline docs/dev/building-on-zksync/useful-address.md:44]:
```suggestion
```

File: docs/dev/building-on-zksync/useful-address.md [hunk 3]
```diff
@@ -56,6 +59,8 @@ _Users should only interact with the token bridge contracts via [zkSync Bridge](
 
 ### Testnet token bridge contracts addresses
 
+These are the addresses that have been deployed and integrated with the token bridge on testnet.
```
Comment (kailash-manasarovar) [inline docs/dev/building-on-zksync/useful-address.md:62]:
```suggestion
```

File: docs/dev/building-on-zksync/useful-address.md [hunk 4]
```diff
@@ -67,11 +72,22 @@ _Users should only interact with the token bridge contracts via [zkSync Bridge](
 
 ## Mainnet known tokens
 
+These are the tokens that are listed on our block explorer. Learn how to [add token](../developer-guides/bridging/bridging-asset.md#add-tokens-to-the-bridge) to our bridge.
```
Comment (kailash-manasarovar) [inline docs/dev/building-on-zksync/useful-address.md:75]:
```suggestion
These tokens are listed on our block explorer. Learn how to [add tokens to the bridge](../developer-guides/bridging/bridging-asset.md#add-tokens-to-the-bridge).
```

File: docs/dev/building-on-zksync/useful-address.md [hunk 5]
```diff
@@ -67,11 +72,22 @@ _Users should only interact with the token bridge contracts via [zkSync Bridge](
...
 | PERP                          | Perpetual     |  [0xbC396689893D065F41bc2C6EcbeE5e0085233447](https://etherscan.io/address/0xbC396689893D065F41bc2C6EcbeE5e0085233447)|[0x42c1c56be243c250AB24D2ecdcC77F9cCAa59601](https://explorer.zksync.io/address/0x42c1c56be243c250AB24D2ecdcC77F9cCAa59601#contract)|
-| USDC                          | USD Coin      | [0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48](https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48)|[0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4](https://explorer.zksync.io/address/0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4#contract)|
+| USDC                          | USD Coin      | [0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48](https://etherscan.io/address/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48)|[0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4](https://explorer.zksync.io/address/0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4#contract)|
```
Comment (kailash-manasarovar) [inline docs/dev/building-on-zksync/useful-address.md:82]: Should this be a verified contract link for USD Coin? I don't think it's correct.

File: docs/dev/building-on-zksync/useful-address.md [hunk 6]
```diff
@@ -56,6 +59,8 @@ _Users should only interact with the token bridge contracts via [zkSync Bridge](
 
 ### Testnet token bridge contracts addresses
```
Comment (kailash-manasarovar) [inline docs/dev/building-on-zksync/useful-address.md:58]:
```suggestion
### Testnet token bridge, contract addresses
```

**Review by kailash-manasarovar — CHANGES_REQUESTED** [review]
Can you check the token addresses. I'm seeing unverified contracts for many of them and it feels incorrect.

**Comment by krofax** [comment]
> Can you check the token addresses. I'm seeing unverified contracts for many of them and it feels incorrect.

If you're referring to L2, not all the tokens are verified, see here - https://explorer.zksync.io/tokenlist

**Comment by kailash-manasarovar** [comment]
> > Can you check the token addresses. I'm seeing unverified contracts for many of them and it feels incorrect.
> 
> If you're referring to L2, not all the tokens are verified, see here - https://explorer.zksync.io/tokenlist

Ah OK, thank you. I'll check again in the a.m. :)

File: docs/dev/building-on-zksync/useful-address.md (hunk 5 above)
Comment (kailash-manasarovar) [reply docs/dev/building-on-zksync/useful-address.md:80]: Is L1 correct?

**Review by kailash-manasarovar — APPROVED** [review]
OK, I trust these are correct .. I was a bit surprised they weren't verified :)

**Pull request merged**